WinUIGallery/MainWindow.xaml

```diff
@@ -1,63 +1,58 @@
-            <!--  TitleBar  -->
             <RowDefinition Height="*" />
-            <!--  NavView  -->
         </Grid.RowDefinitions>
-
-        <Border
-            x:Name="AppTitleBar"
-            Grid.Column="1"
-            Height="{Binding ElementName=NavigationViewControl, Path=CompactPaneLength}"
-            Margin="48,0,0,0"
-            VerticalAlignment="Stretch"
-            AutomationProperties.AutomationId="AppTitleBar"
-            Canvas.ZIndex="1"
-            IsHitTestVisible="True">
-            <StackPanel VerticalAlignment="Stretch" Orientation="Horizontal">
-                <Image
-                    Width="18"
-                    VerticalAlignment="Center"
-                    Source="ms-appx:///Assets/Tiles/BadgeLogo.png" />
-                <TextBlock
-                    x:Name="AppTitle"
-                    Margin="12,0,0,0"
+        <TitleBar
+            x:Name="titleBar"
+            Title="WinUI 3 Gallery"
+            BackRequested="TitleBar_BackRequested"
+            IsBackButtonVisible="{x:Bind rootFrame.CanGoBack, Mode=OneWay}"
+            IsPaneToggleButtonVisible="True"
+            PaneToggleRequested="TitleBar_PaneToggleRequested"
+            Subtitle="Dev">
+            <TitleBar.IconSource>
+                <ImageIconSource ImageSource="/Assets/Tiles/GalleryIcon.ico" />
+            </TitleBar.IconSource>
+            <TitleBar.CenterContent>
+                <AutoSuggestBox
+                    x:Name="controlsSearchBox"
+                    Width="320"
+                    HorizontalAlignment="Stretch"
                     VerticalAlignment="Center"
-                    Style="{StaticResource CaptionTextBlockStyle}"
-                    Text="{x:Bind AppTitleText}" />
-                <TextBox
-                    x:Name="AppTitleBarTextBox"
-                    Height="40"
-                    MinWidth="300"
-                    Margin="16,0,0,0"
-                    PlaceholderText="Enter any text"
-                    Visibility="Collapsed" />
-            </StackPanel>
-        </Border>
-
-        <Grid x:Name="AutomationHelpersPanel" Grid.Column="1">
+                    x:FieldModifier="public"
+                    KeyboardAcceleratorPlacementMode="Hidden"
+                    PlaceholderText="Search"
+                    QueryIcon="Find"
+                    QuerySubmitted="OnControlsSearchBoxQuerySubmitted"
+                    TextChanged="OnControlsSearchBoxTextChanged">
+                    <AutoSuggestBox.KeyboardAccelerators>
+                        <KeyboardAccelerator
+                            Key="F"
+                            Invoked="CtrlF_Invoked"
+                            Modifiers="Control" />
+                    </AutoSuggestBox.KeyboardAccelerators>
+                </AutoSuggestBox>
+            </TitleBar.CenterContent>
+        </TitleBar>
+        <Grid
+            x:Name="AutomationHelpersPanel"
+            Grid.Row="1"
+            Grid.Column="1">
```

Comment:
Both this Grid, and the NavView, use `Column=1`, but there's no ColumnDefinition

Reply:
Thanks, fixed!